README.md

```diff
@@ -9,135 +9,6 @@ A collection of closure models and tools useful for multiphysics codes.
-- `_INSTALL_DIR` points to a directory where the package has been installed by CMake. It expects to find a CMake configuration file in a standard location. If this option is specified and the configuration file cannot be found at that location, the build will produce and error and halt
-- `_IMPORT_DIR` points to a directory that is the top-level of a source-tree of `PKG`. The directory will be imported into the build "as-though" it was an "in-tree" directory
-- using `_IMPORT_DIR` will only check for the existence of a top-level `CMakeLists.txt` file. It does no other evaluation on the directory (e.g. there is no check that this path points to the "correct" package source)
-- `_INSTALL_DIR` and `_IMPORT_DIR` cannot be used for the same package
-
-### Building on Darwin power9 nodes with the IBM xl compiler
-
-```bash
-git clone --recursive https://gitlab.lanl.gov/singularity/singularity-eos.git
-cd singularity-eos
-mkdir bin && cd bin
-source /usr/projects/eap/dotfiles/.bashrc xl reset
-module switch ibm/xlc-16.1.1.5-xlf-16.1.1.5 ibm/xlc-16.1.1.7-xlf-16.1.1.7-gcc-7.4.0
-export FC="xlf_r -F/usr/projects/opt/ppc64le/ibm/xlf-16.1.1.7/xlf/16.1.1/etc/xlf.cfg.rhel.7.8.gcc.7.4.0.cuda.10.1"
-module load cmake/3.17.3
-cd /build/directory
-cmake -DCMAKE_BUILD_TYPE=Release -DSINGULARITY_USE_KOKKOS=ON -DSINGULARITY_USE_HDF5=ON -DSINGULARITY_USE_EOSPAC=ON -DSINGULARITY_EOSPAC_INSTALL_DIR=/usr/projects/eap/spack/spack-develop_20191010/opt/spack/linux-rhel7-power9le/xl_r-16.1.1.7-plain/eospac-6.4.0beta.2-mp6hd267l4iokwr6ght2kp5ukk5zc6vq -DSINGULARITY_BUILD_TESTS=ON ..
-make -j
-make test
-```
-
-## Formatting
-
-This project is formatted with `clang-format` using `clang-format`
-version 12. If contributing, please run clang-format on the files
-you modify.
-
-To run formatting automatically, call `make format_singularity-eos`
-at the build step. Cmake automatically makes a target.
-
-You can also call the bash script `format.sh` in the `utils/scripts`
-directory as
-```bash
-utils/scripts/format.sh
-```
-The script optionally sets the `clang-format` binary from an
-environment variable, e.g.,
-```bash
-CFM=clang-format=12 utils/scripts/format.sh
-```
-
-## Units
-
-singularity-eos, prior to unit modifiers, always uses CGS units internally. For
-concreteness, the internal units of each quantity are:
-
-| Quantity                  | Alternate names      | Units         |
-| ------------------------- | -------------------- | ------------- |
-| Mass density              | rho                  | g cm^-3       |
-| Temperature               | temperature          | K             |
-| Specific Internal Energy  | sie, Internal Energy | erg g^-1      |
-| Pressure                  |                      | erg cm^-3     |
-| Specific Heat             |                      | erg g^-1 K^-1 |
-| Bulk Modulus              |                      | erg cm^-3     |
-| Gruneisen Parameter       |                      |               |
-
```

Comment:
Were these all out of date @Yurlungur ? In general it seems like a good idea to direct people to the docs rather than have to keep a readme and docs up to date.

Reply:
Some of them were not out of date, but I just figured keeping everything in one place is better.